core/project/Dependencies.scala

```diff
@@ -12,8 +12,9 @@ object Version {
   val ScalaOperator = "2.13.3"
   val Spark         = "2.4.5"
   val Flink         = "1.10.0"
-  val KafkaClients  = "2.5.0"
   val EmbeddedKafka = "2.5.0" 
```

Comment:
Can `EmbeddedKafka` go completely now?

Reply:
is still used here:
https://github.com/lightbend/cloudflow/blob/master/core/cloudflow-akka-tests/src/test/scala/cloudflow/akkastream/scaladsl/EmbeddedKafkaSpec.scala

I've spent half an hour port it to Testcontainers but one test started to fail, if you can help I give it another spin and eventually ask for your input, deal?